packages/kit-bg/src/dbs/local/LocalDbBase.ts

```
@@ -4933,27 +4954,67 @@ export abstract class LocalDbBase extends LocalDbBaseContainer {
+        _ctx = await this.txGetContext({ tx });
+        console.log('demoTestTransactionAutoCommit>>>>>>> 3', _ctx);
+
+        await this.txUpdateContext({
+          tx,
+          updater: (r) => {
+            r.backupUUID = `1111: ${new Date().toLocaleTimeString()}`;
+            return Promise.resolve(r);
+          },
+        });
+
+        if (true) throw new OneKeyPlainTextError('test error');
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)